lib/src/inherited_provider.dart

```diff
@@ -564,7 +570,7 @@ abstract class _DelegateState<T, D extends _Delegate<T>> {
 
   bool get hasValue;
 
-  bool debugSetInheritedLock(bool value) {
+  bool debugSetInheritedLock({bool value}) {
```

Comment:
Maybe a **breaking change**.

Reply:
This is not a breaking change. That is only for internal use

Reply:
Using named parameters here does not make sense after thinking about it.